docs/networking.md

```diff
@@ -0,0 +1,262 @@
+# Networking
+Last updated: **December 20, 2023**
+
+## Introduction
+This document breaks down the centralized networking of the AWS Secure Environment Accelerator (ASEA), and best practices for networking on the platform. It aims to make the ASEA's networking easy to grasp, helping anyone working on the platform (and within the guardrails of the BC Gov ASEA) understand how our networking is set up. Good networking is like the backbone of cloud technology. It connects everything, keeps things safe, and helps things run smoothly. Understanding it is crucial for making the most of cloud tools.
+
+
+
+<!-- - Objectives of the document
+- Importance of networking in cloud development and operations -->
+
+## Network architecture
+
+The ASEA network centers around a Transit Gateway, efficiently directing traffic between the internet, and AWS services in various different AWS Accounts on the platform. Networking is broken up into two separate accounts: 
+
+- The Perimeter Account houses our Checkpoint Firewalls which monitor and restrict traffic in and out of the platform. 
+- The Shared Network Account houses the Transit Gateway and acts as a central hub. this account houses all of the VPCs (except for the Perimeter VPC in the Perimeter account) and most other networking resources. These are made available via AWS Resource Access Manager (RAM) to the appropriate accounts based on their OU in the organization. Security Groups are then put in place restrict traffic between resources deployed in the VPCs. All this eliminates the need to create duplicate resources in multiple accounts, reducing the operational overhead of managing those resources in every single account.
+
+The reason for the separation between the Shared Networking and Perimeter accounts is to facilitate networking and security "separation of duties". In summary, the ASEA's networking architecture ensures centralized, organized and secure communication through Transit Gateway routing, separated Perimeter VPC security, and Shared Network resource management as shown below in BC Gov ASEA's complete networking diagram:
+
+![networking-architecture](images/networking/network-architecture.png)
+
```

Comment:
also include the Add the 1.1 Shared VPC diagram from the ASEA docs

Reply:
Added below